metrics/grep.csv

```diff
@@ -0,0 +1,3 @@
+Name,Labels,Description,Child,Initialize
+wmo_wis2_grep_connected_flag,centre_id|report_by,Connection status from GRep to to centre,-,false
```

Comment:
I don't understand that one. Connection will only be to GBs.
We don't have that for GC, GDC,... So ??